### PR TITLE
New profile: singularity (Endgame: Singularity)

### PR DESCRIPTION
**Pull request by NetSysFire**
Tested it myself of course. This is a small, python-based game which needs no network access and is generally quite frugal in terms of resources.

I would deem it fine if this is added to firecfg, as there are little to no configurable options and will work out of the box.

### CODE REVIEW

File: etc/profile-a-l/singularity.profile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+include singularity.local
+include globals.local
```
Comment (rusty-snake) [inline etc/profile-a-l/singularity.profile:2]: Add comments as everywhere.

File: etc/profile-a-l/singularity.profile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+noblacklist ${HOME}/.config/singularity
+noblacklist ${HOME}/.local/share/singularity
```
Comment (rusty-snake) [inline etc/profile-a-l/singularity.profile:5]: Add blacklist to disable-programs.inc

File: etc/profile-a-l/singularity.profile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+net none
+caps.drop all
+disable-mnt
```
Comment (rusty-snake) [inline etc/profile-a-l/singularity.profile:29]: Sort disable-mnt down as per template

File: etc/profile-a-l/singularity.profile [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+nodvd
+noroot
+nou2f
```
Comment (rusty-snake) [inline etc/profile-a-l/singularity.profile:32]:
```suggestion
nou2f

```

File: etc/profile-a-l/singularity.profile (hunk 3 above)
Comment (NetSysFire) [reply etc/profile-a-l/singularity.profile:29]: sort.py did not point this out.

Comment (rusty-snake) [reply etc/profile-a-l/singularity.profile]: sort.py does inline sorting of arguments too `private-*`, `seccomp`, ... but no multiline sorting across the file.

**Comment by kmk3** [comment]
@NetSysFire

Disabled more things in the profile.

Does it still work with the changes?

**Comment by NetSysFire** [comment]
This is a small python game that does not need any networking or even external access. I already thought what I did was already a bit overkill as my initial profile for this was even smaller. There is simply no attack surface to justify additional effort in my opinion.

**Comment by kmk3** [comment]
> This is a small python game that does not need any networking or even
> external access. I already thought what I did was already a bit overkill as
> my initial profile for this was even smaller.

This idea seems backwards.

Why give a program more access than it needs, especially if it is unlikely to
cause problems?

> There is simply no attack surface to justify additional effort in my opinion.

What would be the additional effort?  I mostly just added the rest of the
standard entries from profile.template.

Again, do the changes cause any breakage?

**Comment by NetSysFire** [comment]
I do not have any more motivation to test the new directives. I'd prefer to stick to the things I already tested.

**Comment by netblue30** [comment]
All in, thanks!

**Pull request merged**